### DISCUSSION TITLE
will not activate

### DISCUSSION THREAD

**Discussion by dan71w** [bekriebel/fvtt-module-avclient-livekit#23] [Q&A] [answered]
i have installed this, but when i try to activate on the module management page it wont active, the save module settings button doesn't do anything. any idea what might cause this?

**Comment by bekriebel** [comment, marked as answer]
Hrmm... I'm not sure what could cause that. At that point, the code for the module isn't being run at all. Can you enable / disable other modules? Is there any error reported in the developer console log?

**Reply by dan71w** [reply]
i have disabled all modules, got it to work, now installing modules one at a time, to see where it went weird. but i did get it working i think, at least my camera shows up.

**Reply by bekriebel** [reply]
Great! Yeah, it sounds like some other module is possibly causing a break in the module page.
